api/utils/keypaths/keypaths.go

```diff
@@ -40,8 +40,6 @@ const (
 	fileExtSSHCert = "-cert.pub"
 	// fileExtPPK is the suffix/extension of a file where an SSH keypair is stored in PuTTY PPK format.
 	fileExtPPK = ".ppk"
-	// fileExtPub is the extension of a file where a public key is stored.
```

Comment:
Why is this removed?

Reply:
It's no longer being used, since (currently) this PR removes the public key from the `~/.tsh` key store. We can retrieve the public key directly from the private key instead, or from a signed certificate.

To be clear, I removed it because once I refactored the private key usage, it became clear that this file wasn't actually being used for anything. When we would parse the private key, the public key would just be ignored. This public key would only be used directly if we didn't want to parse the private key to get it, but now we are always parsing the private key up front.

If you'd like, I can add the public key back, which we can use to verify that the private key matches the public key on file, but ultimately it won't do much since we can just check that the private key's public key matches the public key in the certificates.